### ISSUE TITLE
Support for JDK 21

### ISSUE THREAD

**Issue by arcuri82** [alibaba/java-dns-cache-manipulator#252]
Hi,

 thanks for this library. Is there any plan to support JDK 21? I am upgrading a system from 17 to 21, and I get the following error:

```
Caused by: java.lang.ClassNotFoundException: java.net.InetAddress$CachedAddresses
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:421)
        at java.base/java.lang.Class.forName(Class.java:412)
        at com.alibaba.dcm.internal.InetAddressCacheUtilForNew.getConstructorOfInetAddress$CachedAddresses(InetAddressCacheUtilForNew.java:82)
        at com.alibaba.dcm.internal.InetAddressCacheUtilForNew.newCachedAddresses(InetAddressCacheUtilForNew.java:67)
        at com.alibaba.dcm.internal.InetAddressCacheUtilForNew.setInetAddressCache(InetAddressCacheUtilForNew.java:54)
        at com.alibaba.dcm.DnsCacheManipulator.setDnsCache(DnsCacheManipulator.java:46)
        ... 67 more
```

many thanks

**Comment by alexey-anufriev** [comment]
Same problem here. @oldratlee, any plans on this? Thanks in advance!

**Comment by oldratlee** [comment]
@arcuri82 @alexey-anufriev Thanks for your attention. 

Just need to get myself started and take some time to implement. ❤️

**Comment by alexey-anufriev** [comment]
@oldratlee, can I offer you help?
I made a PR for this issue, please take a look https://github.com/alibaba/java-dns-cache-manipulator/pull/265.

**Comment by oldratlee** [comment]
> @oldratlee, can I offer you help? I made a PR for this issue, please take a look #265.

@alexey-anufriev 🚀💕

I'll review the PR tomorrow.

**Comment by alexey-anufriev** [comment]
Thank you!

**Comment by oldratlee** [comment]
@alexey-anufriev @arcuri82 

released [`v1.8.2`](https://github.com/alibaba/java-dns-cache-manipulator/releases/tag/v1.8.2).
